samples/java_layout/build.gradle

```diff
@@ -36,6 +36,15 @@ android {
         viewBinding true
         // BuildConfig is now enabled in sample-app.gradle
     }
+    
+    packaging {
+        resources {
+            excludes += [
+                'META-INF/versions/9/OSGI-INF/MANIFEST.MF',
+                'META-INF/OSGI-INF/MANIFEST.MF'
+            ]
```

Comment:
I see we added this to SDK as well, but will customers still need to add it to their apps too? It'd be great if we could avoid that.